assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -25,7 +25,7 @@
     pass
 
 from feature_importance_metrics.feature_importance_utilities import (
-    compute_categorical_features, convert_pandas_to_spark, log_time_and_message)
+    compute_categorical_features, convert_pandas_to_spark, convert_time_columns, is_categorical_column, log_time_and_message)
```

Comment:
convert_time_columns is missing. Could you add this function?

Reply:
removed thanks!